src/main/resources/templates/editBookstore.html

```diff
@@ -21,7 +21,39 @@ <h2>Upload Book</h2>
         <input type="text" name="publisher"><br><br>
         <label>Genre:</label><br>
         <input type="text" name="genre"><br><br>
-
+        <label>Quantity:</label><br>
+        <select id="quality" form="addBook">
```

Comment:
This says quality instead of I'm assuming quantity. Quantity still 0 for all books uploaded